api/adaptive_load/metric_spec.proto

```diff
@@ -0,0 +1,40 @@
+// Protos for identifying metrics and specifying thresholds.
+
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+import "api/adaptive_load/config.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+
+// Identifies a feedback metric.
+message MetricSpec {
+  // Name of the metric to evaluate. For the set of built-in metric names, see
+  // source/adaptive_load/metrics_plugin_impl.cc. Required.
+  string metric_name = 1 [(validate.rules).string.min_len = 1];
+  // Name of the MetricsPlugin providing the metric ("builtin" for built-in).
```

Comment:
I recommend adopting a convention such as `nighthawk.foo` for builtin extension `foo`. This is what we do in Envoy.

Reply:
Will do.

Reply:
Done